### PR TITLE
444 meta table for semaphore failing

### PR DESCRIPTION
**Pull request by matdenkas**
To test you can run test:
`docker exec semaphore-core python3 -m pytest`

Try running a model:
`docker exec semaphore-core python3 src/semaphoreRunner.py -d TestModels/test_dspec.json`

You can also cause semaphore to fail somewhere.

You can see the results by querying the db:
SELECT * FROM outputs
JOIN model_runs ON outputs."id" = model_runs."outputID"

This PR contains a DB migration script, the number might need to be changed on merge

### CODE REVIEW

File: tools/DatabaseMigration/2_6/2_6_DatabaseMigration.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#----------------------------------
+# 
+#
+#Imports
+from DatabaseMigration.IDatabaseMigration import IDatabaseMigration
+from sqlalchemy import Table, Column, Integer, String, DateTime, MetaData, UniqueConstraint, Engine, ForeignKey, Boolean, Interval, text
+from sqlalchemy.dialects.postgresql import insert
+import csv
+
+CSV_FILE_PATHS = './tools/DatabaseMigration/1_0/init_data'
```
Comment (lovelysandlonelys) [inline tools/DatabaseMigration/2_6/2_6_DatabaseMigration.py:21]: CSV file path 1_0! I think you can just fully remove this line.

File: tools/DatabaseMigration/2_6/2_6_DatabaseMigration.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+#Implementing methods of interface
+class Migrator(IDatabaseMigration):
+
+    def update(self, databaseEngine: Engine) -> bool:
+        """This function updates the database to version 1.0 which includes the base schema and data locations.
```
Comment (lovelysandlonelys) [inline tools/DatabaseMigration/2_6/2_6_DatabaseMigration.py:27]: incorrect function documentation 😭

**Review by lovelysandlonelys — CHANGES_REQUESTED** [review]
image
Tests Passed
Just tiny documentation changes, everything else looks right to me.

File: tools/DatabaseMigration/2_6/2_6_DatabaseMigration.py (hunk 2 above)
Comment (lovelysandlonelys) [reply tools/DatabaseMigration/2_6/2_6_DatabaseMigration.py:27]: ?? still here

Comment (matdenkas) [reply tools/DatabaseMigration/2_6/2_6_DatabaseMigration.py]: SO SO SO MY BAD! Didn't push, sorry for the extra ping

File: src/semaphoreRunner.py [hunk 3]
```diff
@@ -37,37 +91,37 @@ def run_semaphore(fileName: str, executionTime: datetime = None, toss: bool = Fa
             if executionTime is None: 
                 executionTime = datetime.now()
 
-            IG = InputGatherer(fileName)
-            model_name = IG.get_dspec().modelName
-            MW = ModelWrapper(IG)
+            input_gatherer = InputGatherer(fileName)
```
Comment (abeasley1722) [inline src/semaphoreRunner.py:94]: Good improvement to the variable name

**Comment by abeasley1722** [comment]
Tests Passed

**Review by lovelysandlonelys — APPROVED** [review]
It's looking like this will go in first so like joy said, might have to change to 2.5 but otherwise 👍🏻

**Pull request merged**